data/templates/frr/bgp.frr.tmpl

```diff
@@ -58,6 +58,10 @@
 {%   if config.ttl_security is defined and config.ttl_security.hops is defined and config.ttl_security.hops is not none %}
  neighbor {{ neighbor }} ttl-security hops {{ config.ttl_security.hops }}
 {%   endif %}
+{%   if config.timers is defined and config.timers.connect is defined and config.timers.holdtime is defined and config.timers.keepalive is defined %}
```

Comment:
With this `{% if ... %}` block one can only have a BGP connect timer when also a holdtime and keepalive is defined. FRR supports setting the BGP connect time independently from the holdtime and keepalive.

Please make BGP connect time configurable independently from holdtime and keepalive.

Reply:
@c-po done

Reply:
This is a bit too complicated/too many nesting levels. Please shorten this to:

```
{%   if config.timers is defined %}
{%     if config.timers.connect is defined and config.timers.connect is not none %}
 neighbor {{ neighbor }} timers connect {{ config.timers.connect }}
{%     endif %}
{%     if config.timers.holdtime is defined and config.timers.keepalive is defined and config.timers.holdtime is not none and config.timers.keepalive is not none %}
 neighbor {{ neighbor }} timers {{ config.timers.keepalive }} {{ config.timers.holdtime }}
{%     endif %}
{%   endif %}
```

Reply:
Done